app/jobs/seekr.rb

```diff
@@ -80,13 +90,18 @@ def self.perform(slug, searchid)
 
     Dashing.send_event("#{slug}_twitter_mentions", {:comments => mentions}, :cache => true)
 
+    # Blacklist of Hashtags
+
+    blacklist = JSON.parse(File.read(Rails.root.join("app/jobs/hashtags.json")))['hashtags']
```

Comment:
Cara, e se o arquivo não existir? Vai quebrar sem explicação? That's no good man.